<ui version="4.0" >
 <class>TimeLineViewClass</class>
 <widget class="QWidget" name="TimeLineViewClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TimeLineView</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButton" >
       <property name="text" >
        <string>Curve</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_CurrentFrameNo" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="RulerWidget" native="1" name="widgetRuler" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_minSceneFrameNo" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="RangeWidget" native="1" name="widgetRange" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_maxSceneFrameNo" >
       <property name="text" >
        <string>  0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="toolButtonHead" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet" >
          <string>border: none</string>
         </property>
         <property name="text" >
          <string>Head</string>
         </property>
         <property name="icon" >
          <iconset resource="timelineview.qrc" >:/TimelineView/../../resources/button_timeline_head.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="popupMode" >
          <enum>QToolButton::DelayedPopup</enum>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonPrevframe" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text" >
          <string>Prev. Frame</string>
         </property>
         <property name="icon" >
          <iconset>../../resources/button_timeline_prevframe.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonBack" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>25</width>
           <height>20</height>
          </size>
         </property>
         <property name="text" >
          <string>Backward</string>
         </property>
         <property name="icon" >
          <iconset>../../resources/button_timeline_back.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>25</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonPause" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text" >
          <string>Pause</string>
         </property>
         <property name="icon" >
          <iconset>../../resources/button_timeline_pause.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonPlay" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>25</width>
           <height>20</height>
          </size>
         </property>
         <property name="text" >
          <string>Play</string>
         </property>
         <property name="icon" >
          <iconset>../../resources/button_timeline_play.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>25</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonNextframe" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text" >
          <string>Next Frame</string>
         </property>
         <property name="icon" >
          <iconset>../../resources/button_timeline_nextframe.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonEnd" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="baseSize" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text" >
          <string>End</string>
         </property>
         <property name="icon" >
          <iconset>../../resources/button_timeline_end.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTake" >
       <property name="minimumSize" >
        <size>
         <width>93</width>
         <height>0</height>
        </size>
       </property>
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>160</red>
             <green>160</green>
             <blue>160</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>160</red>
             <green>160</green>
             <blue>160</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>236</red>
             <green>233</green>
             <blue>216</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxSPD" >
       <property name="maximumSize" >
        <size>
         <width>62</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="Light" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>160</red>
             <green>160</green>
             <blue>160</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Light" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>160</red>
             <green>160</green>
             <blue>160</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Light" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>236</red>
             <green>233</green>
             <blue>216</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxFPS" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>62</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>160</red>
             <green>160</green>
             <blue>160</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>160</red>
             <green>160</green>
             <blue>160</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>236</red>
             <green>233</green>
             <blue>216</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>RangeWidget</class>
   <extends>QWidget</extends>
   <header>rangewidget.h</header>
  </customwidget>
  <customwidget>
   <class>RulerWidget</class>
   <extends>QWidget</extends>
   <header>rulerwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="timelineview.qrc" />
  <include location="timelineview.qrc" />
 </resources>
 <connections/>
</ui>
